extension/templates/devTools/activity/graph.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  <h4>Activity Graph</h4>
```

Comment:
Maybe this should also have an element for the visualization to go in. It could be helpful to have space for more things like an axis or "summary" graph later too